numpyro/distributions/util.py

```diff
@@ -395,6 +282,103 @@ def multinomial_rvs(key, n, p, shape=()):
     return np.reshape(samples_2D, shape + p.shape[-1:]) - excess
 
 
+def multinomial(key, p, n, shape=()):
+    return _multinomial(key, p, n, shape)
+
+
+def _xlogy_jvp_lhs(g, x, y):
```

Comment:
I group all samplers to one place, so other utilities are grouped into one place too.